openhtf/io/frontend/__init__.py

```diff
@@ -170,9 +170,9 @@ def _discover(self):
           if (hostport, test_uid) not in self._watchers:
             self._watchers[hostport, test_uid] = TestWatcher(
                 hostport, remote_test,self._on_update_callback)
-      except (socket.error, station_api.StationUnreachableError) as e:
+      except station_api.StationUnreachableError:
         _LOG.debug('Station at %s is unreachable.', hostport)
-
+    
```

Comment:
madsci and you send PRs that add and remove trailing whitespace and it makes the diffs messy. Can you make sure your editor strips trailing whitespace automatically?